docs/patterns/_matrix.md

```diff
@@ -0,0 +1,120 @@
+---
+collection: patterns
+title: Matrix
+---
+
+<ul class="p-matrix clearfix">
+    <li class="p-matrix__item">
+        <div class="p-matrix__img">
+          <img src="http://placehold.it/60x60" alt="icon">
+        </div>
+        <div class="p-matrix__content">
+            <h3 class="p-matrix__title"><a class="p-matrix__link" href="#">Title</a></h3>
+            <p class="p-matrix__desc">Short description</p>
```

Comment:
Do you think desc is nicer then description? I like to be clear over saving bytes?